listenbrainz/db/dump.py

```diff
@@ -46,6 +46,7 @@
 MAIN_FTP_SERVER_URL = "ftp.eu.metabrainz.org"
 FULLEXPORT_MAX_AGE = 17  # days
 INCREMENTAL_MAX_AGE = 26  # hours
+FEEDBACK_MAX_AGE = 9  # days
```

Comment:
looking at the size of a feedback dump, it's pretty small and so should dump really quickly. maybe 8 days?

Reply:
I had chosen 9 origianally because the check_dumps scripts run only once daily so was unsure whether the time difference of 8 days would be sufficient. I  now checked manually, it is. So changed to 8.